src/components/com_tjdashboard/administrator/libraries/dashboard.php

```diff
@@ -156,9 +156,8 @@ public function save()
 
 			return false;
 		}
-		$this->dashboard_id = $table->dashboard_id;
 
-		return $this->dashboard_id;
+		return $table->dashboard_id;
```

Comment:
@monikaatvowel This is not required. Can you assign this property to object itself?